volatility3/framework/plugins/windows/strings.py

```diff
@@ -81,22 +87,45 @@ def _generator(self) -> Generator[Tuple, None, None]:
         last_prog: float = 0
         line_count: float = 0
         num_strings = len(string_list)
-        for offset, string in string_list:
+        for phys_offset, string in string_list:
             line_count += 1
-            try:
-                revmap_list = [
-                    name + ":" + hex(offset) for (name, offset) in revmap[offset >> 12]
-                ]
-            except (IndexError, KeyError):
-                revmap_list = ["FREE MEMORY"]
-            yield (
-                0,
-                (
-                    str(string, "latin-1"),
-                    format_hints.Hex(offset),
-                    ", ".join(revmap_list),
-                ),
+
+            # We should really take care of this in the revmap generator
```

Comment:
This kind of comment makes me feel like it's a rushed patch...  5;P  If it should be done there, is there a good reason it's being done here instead?

Reply:
This used to be a `", ".join(revmap_list),` i would have to rewrite the entire revmap parser and i have to be honest i dont know enough about what thats doing so this felt like a safer option